jax/_src/scipy/stats/uniform.py

```diff
@@ -32,3 +33,22 @@ def logpdf(x: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
 @_wraps(osp_stats.uniform.pdf, update_doc=False)
 def pdf(x: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
   return lax.exp(logpdf(x, loc, scale))
+
+@_wraps(osp_stats.uniform.cdf, update_doc=False)
+def cdf(x: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
+  x, loc, scale = promote_args_inexact("uniform.cdf", x, loc, scale)
+  zero, one = jnp.array(0, x.dtype), jnp.array(1, x.dtype)
+  conds = [lax.lt(x, loc), lax.gt(x, lax.add(loc, scale)), lax.ge(x, loc) & lax.le(x, lax.add(loc, scale))]
+  vals = [zero, one, lax.div(lax.sub(x, loc), scale)]
+
+  return jnp.select(conds, vals)
+
+@_wraps(osp_stats.uniform.ppf, update_doc=False)
+def ppf(q: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
+  q, loc, scale = promote_args_inexact("uniform.ppf", q, loc, scale)
+  zero, one = jnp.array(0, q.dtype), jnp.array(1, q.dtype)
+  return where(
+    jnp.isnan(q) | (q < zero) | (q > one),
```

Comment:
You can use `(q < 0) | (q > 1)` here, because this is not a context where strict dtype matching is required.